code/game/mecha/mecha.dm

```diff
@@ -145,7 +146,7 @@
 /obj/mecha/get_cell()
 	return cell
 
-/obj/mecha/Destroy()
+/obj/mecha/deconstruct(disassembled = TRUE)

```

Comment:
That may be but Destroy does a lot of cleanup that will be needed if deconstruct isn't called. Instead just make a var that determines whether or not a mech will leave debris and set it to FALSE on export